### DISCUSSION TITLE
I am new to ECL.Trying to install ECL IDE. Facing problem with preference setting IP address.Which IPaddress should i specify and oe to find out?

### DISCUSSION THREAD

**Discussion by manonmanis471** [hpcc-systems/vscode-ecl#336] [General]
I am new to ECL.Trying to install ECL IDE. Facing problem with preference setting IP address.Which IPaddress should i specify and oe to find out?

**Comment by GordonSmith** [comment]
Hi - This repo is for the VS Code ECL Extension.  In general the IP address you enter is the IP address for the HPCC Platform instance you want to target (typically the same IP as ECL Watch).

If you don't have a HPCC Platform instance, you can leverage GitPod to launch a web version of the ECL Extension in VS Code targeting a 1 Node cloud environment by clicking this link:  https://gitpod.io/#https://github.com/GordonSmith/ECL-Playground see https://github.com/GordonSmith/ECL-Playground for more details...

**Comment by manonmanis471** [comment]
Thank you very much sir.Let me try in this regards.
*With Regards,*



*Manonmani.SAssistant Professor,*

*Department of Computer Science and Engineering,RVCE*
